pkg/container/qemu_runner.go

```diff
@@ -1083,6 +1088,30 @@ func sendSSHCommand(ctx context.Context, user, address string,
 	return nil
 }
 
+func getUserSSHKey() ([]byte, error) {
+	socket := os.Getenv("SSH_AUTH_SOCK")
+	conn, err := net.Dial("unix", socket)
```

Comment:
What if this isn't set?

Reply:
I have this set to empty string...

Reply:
This did cause issues in CI: https://github.com/chainguard-dev/melange/actions/runs/15114937303/job/42483172441?pr=1994#step:16:87

Reply:
it will not find an ssh agent to talk to
another approach would be to naively list `~/.ssh/*.pub` and add those

Reply:
Fixed, now it will check the socket, if fails, fallback to check for files, else again just return nil and will skip the injection